crates/config/src/lib.rs

```diff
@@ -399,8 +400,11 @@ pub struct Config {
 pub static STANDALONE_FALLBACK_SECTIONS: Lazy<HashMap<&'static str, &'static str>> =
     Lazy::new(|| HashMap::from([("invariant", "fuzz")]));
 
-/// Deprecated keys.
-pub static DEPRECATIONS: Lazy<HashMap<String, String>> = Lazy::new(|| HashMap::from([]));
+/// Deprecated keys and their replacements.
+///
+/// See [Warning::DeprecatedKey]
+pub static DEPRECATIONS: Lazy<HashMap<String, String>> =
```

Comment:
Since there aren't / there won't be many, this should just be a `&[(&str, &str)]` with a linear search where it's used.

Reply:
right, unrelated to this pr but might as well just fix that here